ryardley/pdsl
# Fix babel plugin bugs

Babel plugin brought up to date with the new changes to PDSL v4. 

* Test generated code via eval
* As new literals are created tests must be updated
* Use object methods instead of destructuring to simplify code
* Add some new syntax to assertions to ensure plugin works well with new syntax
